src/components/siteSelection/SiteSelection.vue

```diff
@@ -85,7 +86,7 @@ export default defineComponent({
       useWaterQualityStore
     ),
     availableSites() {
-      return this.searchParameterStore.availableSites
+      return this.searchParameterStore.availableSites.sort((s1, s2) => alphabeticCompare(s1.name, s2.name))
```

Comment:
Performing the ordering after each batch concatenation was slow, so moved the ordering here, where it matters more and did not cause such issues